data/sheragi/sheragi outfits.txt

```diff
@@ -416,3 +416,28 @@ outfit "Heavy Ion Fragment"
 		"hit force" 4.5
 		"shield damage" 1.79
 		"hull damage" 1.47
+
+outfit "Retrofitted Fusion Afterburner"
+	category "Engines"
+	cost 400000
+	thumbnail "outfit/fusion afterburner"
+	"mass" 70
+	"outfit space" -70
+	"engine capacity" -70
+	"afterburner thrust" 430
+	"afterburner energy" 2.3
+	"afterburner fuel" 1.1
+	"afterburner heat" 110
```

Comment:
```suggestion
	"engine capacity" -59
	"afterburner thrust" 378
	"afterburner energy" -12.2
	"afterburner fuel" 1.1
	"afterburner heat" 110
	"unique" 1
```
Some thoughts for making it more analogous to the normal Sheragi engines, a source of energy and thrust, also the unique attribute.